components/org.apache.stratos.common/src/test/java/org/apache/stratos/common/test/publisher/ThriftClientConfigParserTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*  http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied.  See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.apache.stratos.common.test.publisher;
+
+import junit.framework.TestCase;
+import org.apache.stratos.common.statistics.publisher.ThriftClientConfig;
+import org.junit.Test;
+
+/**
+ * Thrift Client configuration.
+ *
+ * @author  supunr
+ * @contact supunr@wso2.com
+ * @date    1/27/15
+ */
+public class ThriftClientConfigParserTest extends TestCase {
+    /**
+     * Test ThriftClientConfigParser parse method to check whether it
+     * reads the correct xml filed values.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testName() throws Exception{
```

Comment:
May be we need to use a sample configuration file to write the unit test. Are we using the default values here?